### ISSUE TITLE
extension.cscope-code.*.input not enabled

### ISSUE THREAD

**Issue by zxbuaa** [SeungukShin/cscope-code#4]
I updated to v0.0.4 and found the commands no longer support manually input symbol to search, and only allows search symbol from current cursor (and no Enter to confirm needed which is great). And it seems you added another batch of commands (such as extension.cscope-code.symbol.input) to support manually input. However, these commands do not appear in Ctrl+Shift+P. 

Name | Description | Keyboard Shortcuts | Menu Contexts
-----|-------------|--------------------|--------------
extension.cscope-code.build | Cscope Code: Build | Ctrl+.B |  
extension.cscope-code.symbol | Cscope Code: Symbol | Ctrl+.S |  
extension.cscope-code.definition | Cscope Code: Definition | Ctrl+.G |  
extension.cscope-code.callee | Cscope Code: Callee | Ctrl+.A |  
extension.cscope-code.caller | Cscope Code: Caller | Ctrl+.C |  
extension.cscope-code.text | Cscope Code: Text | Ctrl+.T |  
extension.cscope-code.egrep | Cscope Code: Pattern | Ctrl+.E |  
extension.cscope-code.file | Cscope Code: File | Ctrl+.F |  
extension.cscope-code.include | Cscope Code: Include | Ctrl+.I |  
extension.cscope-code.set | Cscope Code: Set | Ctrl+.N |  
extension.cscope-code.result | Cscope Code: Result | Ctrl+.. |  
extension.cscope-code.pop | Cscope Code: Pop | Ctrl+.O |  
extension.cscope-code.symbol.input |   | Ctrl+.Shift+S |  
extension.cscope-code.definition.input |   | Ctrl+.Shift+G |  
extension.cscope-code.callee.input |   | Ctrl+.Shift+A |  
extension.cscope-code.caller.input |   | Ctrl+.Shift+C |  
extension.cscope-code.text.input |   | Ctrl+.Shift+T |  
extension.cscope-code.egrep.input |   | Ctrl+.Shift+E |  
extension.cscope-code.file.input |   | Ctrl+.Shift+F |  
extension.cscope-code.include.input |   | Ctrl+.Shift+I |  
extension.cscope-code.set.input |   | Ctrl+.Shift+N

**Comment by SeungukShin** [comment]
Dear Xiaobo,
Sorry for inconvenience.
I added commands which you can modify the search word and released v0.0.5.
Thank you.

**Comment by zxbuaa** [comment]
Thank you Shin for the prompt fix! I updated v0.0.5 and the commands work. With one exception that extension.cscope-code.egrep.input seems not work, it yields below error when I type regex. (with no error log in cscope or output panel.) If you have time you may have a try.
```
Command 'Cscope Code: Pattern' resulted in an error (An unknown error occurred. Please consult the log for more details.)
```

**Comment by SeungukShin** [comment]
Dear Xiaobo,
Thank you for your feedback.
The error you mentioned occurs when QuickPick appears but no entry is selected (accepted).
I fixed this issue with e3ac198a. I'll release it with other changes soon or later.
Meanwhile, you can use the following package.
[cscope-code-0.0.5.zip](https://github.com/SeungukShin/cscope-code/files/5212328/cscope-code-0.0.5.zip)
Thank you.
